packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -236,17 +235,25 @@ const AccountWizard = props => {
         return properties;
     };
 
-    const submitIDVData = async (document_type, document_number, country_code) => {
-        const idv_submit_data = {
-            identity_verification_document_add: 1,
-            document_number,
-            document_type: document_type.id,
-            issuing_country: country_code,
-        };
+    const submitIDVData = async (document_type, document_number, document_additional, country_code) => {
```

Comment:
You can give `document_additional` a default value and remove all the conditions to check if it is undefined or not.